experimental/FTheoryTools/src/HypersurfaceModels/attributes.jl

```diff
@@ -336,11 +336,12 @@ end
 
 
 @doc raw"""
-    singular_loci(h::HypersurfaceModel)
+    singular_loci(h::HypersurfaceModel; rand_seed::AbstractRNG = Random.default_rng())
 
 Return the singular loci of the Weierstrass model equivalent to the given hypersurface model,
 along with the order of vanishing of ``(f, g, \Delta)`` at each locus and the corresponding
-refined Tate fiber type. See [singular_loci(w::WeierstrassModel)](@ref) for more details.
+refined Tate fiber type. See [singular_loci(w::WeierstrassModel; rand_seed::AbstractRNG = Random.default_rng())](@ref)
```

Comment:
```suggestion
refined Tate fiber type. See [`singular_loci(w::WeierstrassModel)`](@ref)
```
this is missing backticks.
and also you can drop all kwargs from references, as they are not used in dispatch (thus reducing the diff and making the docs more future-proof (wrong kwargs are even worse for readers of the md))

(applicable in more places in this PR)

Reply:
Thank you. Fixed.